eslint.config.js

```diff
@@ -0,0 +1,97 @@
+import hypothesis from 'eslint-config-hypothesis';
+import jsxA11y from 'eslint-plugin-jsx-a11y';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  {
+    ignores: [
+      '.tox/**/*',
+      '.yalc/**/*',
+      '.yarn/**/*',
+      'build/**/*',
+      '**/vendor/**/*.js',
+      '**/coverage/**/*',
+      'docs/_build/*',
+      'dev-server/static/**/*.js',
+    ],
```

Comment:
Many of these files are already ignored by `.gitignore`. Does ESLint not automatically respect that?

Reply:
No, doesn't seem to. I even had to add a few more there that were not part of `.eslintignore` and were getting scanned now (`.tox`, `.yarn` and `.yalc` specifically).